apps/hannk/util/model_runner.cpp

```diff
@@ -617,7 +617,13 @@ void ModelRunner::run(const std::string &filename) {
     // ----- Log benchmark times
     if (do_benchmark) {
         for (WhichRun i : active_runs) {
-            std::cout << RunNames[i] << " Time: " << std::chrono::duration_cast<std::chrono::microseconds>(results[i].time).count() << " us\n";
+            const auto t = std::chrono::duration_cast<std::chrono::microseconds>(results[i].time).count();
+            std::cout << RunNames[i] << " Time: " << t << " us";
+            if (i != kTfLite) {
+                const auto t_ref = std::chrono::duration_cast<std::chrono::microseconds>(results[kTfLite].time).count();
+                std::cout << " (Speedup: " << std::fixed << std::setprecision(2) << (double)t_ref / (double)t << ")";
```

Comment:
Oh missed this, can we also not do this? Again, the reason being, copy/pasting into a spreadsheet is a lot more annoying, and actually just calculating this in the spreadsheet is easy.

I think in general we should just not change the format of the output in this (unrelated) PR.

Reply:
Well, sure, but it requires you actually copy/paste stuff into a spreadsheet, which is annoying (to me) for quick checks.

I'll back out these changes here and propose them elsewhere, and we can discuss our different approaches to this so we aren't competing to optimize for different paths...